idp.merge.properties still needed?

New to the Shibboleth IdP world thankfully I found your tool and gave it a go on a clean host:

git clone
./fetch-jetty
./fetch-shib
Adjust SCOPE and HOST in install-idp
./install

I got:

`BUILD FAILED
/data/shibboleth-dist/bin/build.xml:20: net.shibboleth.utilities.java.support.logic.ConstraintViolationException: property 'idp.sealer.keyPassword' cannot be replaced
	at net.shibboleth.utilities.java.support.logic.Constraint.isFalse(Constraint.java:63)
	at net.shibboleth.idp.installer.PropertiesWithComments.replaceProperty(PropertiesWithComments.java:235)
	at net.shibboleth.idp.installer.PropertiesWithComments.replaceProperties(PropertiesWithComments.java:221)
	at net.shibboleth.idp.installer.V4Install.populatePropertyFiles(V4Install.java:249)
	at net.shibboleth.idp.installer.V4Install.execute(V4Install.java:106)
	at net.shibboleth.idp.installer.ant.impl.V4InstallTask.execute(V4InstallTask.java:95)
	at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:292)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:99)
	at org.apache.tools.ant.Task.perform(Task.java:350)
	at org.apache.tools.ant.Target.execute(Target.java:449)
	at org.apache.tools.ant.Target.performTasks(Target.java:470)
	at org.apache.tools.ant.Project.executeSortedTargets(Project.java:1391)
	at org.apache.tools.ant.Project.executeTarget(Project.java:1364)
	at org.apache.tools.ant.helper.DefaultExecutor.executeTargets(DefaultExecutor.java:41)
	at org.apache.tools.ant.Project.executeTargets(Project.java:1254)
	at org.apache.tools.ant.Main.runBuild(Main.java:830)
	at org.apache.tools.ant.Main.startAnt(Main.java:223)
	at org.apache.tools.ant.Main.start(Main.java:190)
	at org.apache.tools.ant.Main.main(Main.java:274)`

After a bit of googling I found [https://issues.shibboleth.net/jira/browse/IDP-1556 ](https://issues.shibboleth.net/jira/browse/IDP-1556) which hints that IdP v4 no longer supports idp.sealer.storePassword and idp.sealer.keyPassword in idp.merge.properties. And indeed after starting from scratch and removing those entries the build succeeded.

Is idp.merge.properties still needed with v4?

--- a/install-idp
+++ b/install-idp
@@ -44,7 +44,6 @@ echo \# Properties controlling the installation of the Shibboleth IdP>$I
 #
 $DIST/bin/install.sh \
     -Didp.property.file=idp.install.properties \
-    -Didp.merge.properties=idp.merge.properties \
     -Didp.src.dir=$DIST \
     -Didp.target.dir=$IDP_HOME \
     -Didp.scope=$SCOPE \

--- a/install-idp
+++ b/install-idp
@@ -32,12 +32,6 @@ echo \# Properties controlling the installation of the Shibboleth IdP>$I
 # Uncomment the following to preserve the property files
 # echo idp.no.tidy=true>>$I
 
-S=$DIST/idp.merge.properties
-echo idp.scope=$SCOPE>>$S
-echo idp.entityID=$ENTITYID>>$S
-echo idp.sealer.storePassword=$SEALPASS>>$S
-echo idp.sealer.keyPassword=$SEALPASS>>$S
-
 #
 # Perform the Shibboleth IdP install process.
 #

--- a/install-idp
+++ b/install-idp
@@ -47,6 +47,7 @@ $DIST/bin/install.sh \
     -Didp.src.dir=$DIST \
     -Didp.target.dir=$IDP_HOME \
     -Didp.scope=$SCOPE \
+    -Didp.entityID=$ENTITYID \
     -Didp.host.name=$HOST \
     -Didp.sealer.password=$SEALPASS \
     -Didp.keystore.password=$UFPASS \

